From 5947b17ec9da6651b22a060854b825114ec22ddb Mon Sep 17 00:00:00 2001
From: Bernd Machenschalk <bernd.machenschalk@ligo.org>
Date: Sat, 6 Feb 2016 01:24:18 +0100
Subject: [PATCH 3/7] E@H hack: always use dumb shelve

---
 scipy/weave/catalog.py | 13 ++-----------
 1 file changed, 2 insertions(+), 11 deletions(-)

diff --git a/scipy/weave/catalog.py b/scipy/weave/catalog.py
index 85b6c8a..fd58c7c 100644
--- a/scipy/weave/catalog.py
+++ b/scipy/weave/catalog.py
@@ -41,17 +41,8 @@ import tempfile
 import warnings
 import traceback
 
-try:
-    # importing dbhash is necessary because this regularly fails on Python 2.x
-    # installs (due to known bsddb issues).  While importing shelve doesn't
-    # fail, it won't work correctly if dbhash import fails.  So in that case we
-    # want to use _dumb_shelve
-    import dbhash
-    import shelve
-    dumb = 0
-except ImportError:
-    from . import _dumb_shelve as shelve
-    dumb = 1
+from . import _dumb_shelve as shelve
+dumb = 1
 
 
 def getmodule(object):
-- 
2.6.2

