From 15642a1b91d436d8a78b57c1850cf26249c94e32 Mon Sep 17 00:00:00 2001
From: Bernd Machenschalk <bernd.machenschalk@ligo.org>
Date: Mon, 22 Feb 2016 12:12:11 +0100
Subject: [PATCH 6/7] E@H hack: _dumbdb: open files in binary mode

---
 scipy/weave/_dumbdbm_patched.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/scipy/weave/_dumbdbm_patched.py b/scipy/weave/_dumbdbm_patched.py
index 8bb5615..e47d127 100644
--- a/scipy/weave/_dumbdbm_patched.py
+++ b/scipy/weave/_dumbdbm_patched.py
@@ -42,9 +42,9 @@ class _Database(object):
         self._bakfile = file + '.bak'
         # Mod by Jack: create data file if needed
         try:
-            f = _open(self._datfile, 'r')
+            f = _open(self._datfile, 'rb')
         except IOError:
-            f = _open(self._datfile, 'w')
+            f = _open(self._datfile, 'wb')
         f.close()
         self._update()
 
@@ -73,7 +73,7 @@ class _Database(object):
             _os.rename(self._dirfile, self._bakfile)
         except _os.error:
             pass
-        f = _open(self._dirfile, 'w')
+        f = _open(self._dirfile, 'wb')
         for key, (pos, siz) in self._index.items():
             f.write("%s, (%s, %s)\n" % (repr(key), repr(pos), repr(siz)))
         f.close()
-- 
2.6.2

