From acd92a9ada201a3d0092dc1f7621d138045c1ff2 Mon Sep 17 00:00:00 2001
From: Bernd Machenschalk <bernd.machenschalk@ligo.org>
Date: Mon, 22 Feb 2016 14:31:58 +0100
Subject: [PATCH 7/7] E@H hack: _dumb_shelve: use pickle ASCII format

---
 scipy/weave/_dumb_shelve.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scipy/weave/_dumb_shelve.py b/scipy/weave/_dumb_shelve.py
index d97ebf4..ade175d 100644
--- a/scipy/weave/_dumb_shelve.py
+++ b/scipy/weave/_dumb_shelve.py
@@ -26,7 +26,7 @@ class DbfilenameShelf(Shelf):
         return pickle.loads(r)
 
     def __setitem__(self, key, value):
-        self.dict[key] = pickle.dumps(value,1)
+        self.dict[key] = pickle.dumps(value)
 
 
 def open(filename, flag='c'):
-- 
2.6.2

