From 4bc1cec3e17998818fb0fb64191e7c5173b1b3c7 Mon Sep 17 00:00:00 2001
From: Bernd Machenschalk <bernd.machenschalk@ligo.org>
Date: Mon, 7 Mar 2016 10:54:01 +0100
Subject: [PATCH 3/5] LALFrameIO: fix parsing URLs with whitespaces

---
 lalframe/src/LALFrameIO.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lalframe/src/LALFrameIO.c b/lalframe/src/LALFrameIO.c
index ccee64e..eac89b9 100644
--- a/lalframe/src/LALFrameIO.c
+++ b/lalframe/src/LALFrameIO.c
@@ -79,7 +79,7 @@ LALFrFile *XLALFrFileOpenURL(const char *url)
     XLAL_CHECK_NULL(strlen(url) < FILENAME_MAX, XLAL_EBADLEN,
         "url %s is too long", url);
 
-    n = sscanf(url, "%[^:]://%[^/]%s", prot, host, path);
+    n = sscanf(url, "%[^:]://%[^/]%[^\t\n]", prot, host, path);
     if (n != 3) {       /* perhaps the hostname has been omitted */
         XLALStringCopy(host, "localhost", sizeof(host));
         if (n != 2) {   /* assume the whole thing is a file path */
-- 
2.6.2

